File: cronometro.cpp
int main() {
    int segundos = -1;
    int minutos = 0;
    int inicio = 1;

    while (inicio == 1) {
        segundos++;
        // Pausa por 1 segundo
        for (int i = 0; i < 100000000; i++);  // Aproximação de um delay, não é exato

        if (segundos == 60) {
            minutos++;
            segundos = 0;
        }

        if (segundos <= 9) {
            // Imprime com um zero à esquerda
            printf("%d:0%d\n", minutos, segundos);
        } else {
            // Imprime normalmente
            printf("%d:%d\n", minutos, segundos);
        }
    }

    return 0;
}

File: cronometro.py
import time

segundos = -1
minutos = 0

inicio = 1

while inicio == 1:
    segundos = segundos + 1
    time.sleep(1)
    if segundos == 60:
        minutos = minutos + 1
        segundos = 0
    if segundos <= 9:
        print(f"{minutos}:0{segundos}")
    else:
        print(f"{minutos}:{segundos}")
    